LiteCore/tests/c4BaseTest.cc

```diff
@@ -144,6 +146,30 @@ TEST_CASE("C4Error Reporting Macros", "[Errors][C]") {
 }
 
 
+TEST_CASE_METHOD(C4Test, "Database Flag FullSync", "[Database][C]") {
```

Comment:
There should probably at least be a manual inspection of one of these databases to confirm that the full sync is taking effect at the SQLite setting.

Reply:
I did manual inspection here at 
`            _exec(format("PRAGMA cache_size=%d; "            // Memory cache
                         "PRAGMA mmap_size=%d; "             // Memory-mapped reads
                         "PRAGMA synchronous=%s; "           // Speeds up commits
                         "PRAGMA journal_size_limit=%lld; "  // Limit WAL disk usage
                         "PRAGMA case_sensitive_like=true; "   // Case sensitive LIKE, for N1QL compat
                         "PRAGMA fullfsync=ON",              // Attempt to mitigate damage due to sudden loss of power (iOS / macOS)
                         -(int)kCacheSize/1024, kMMapSize, getOptions().diskSyncFull?"full":"normal",
                         (long long)kJournalSize));
`
by breaking there and inspected the string in either case constructed by "format", particularly, "full" vs "normal".

Reply:
Is it sufficient?

Reply:
That seems reasonable